src/overlays/actors/ovl_En_Ssh/z_en_ssh.c

```diff
@@ -4,12 +4,54 @@
 
 #define THIS ((EnSsh*)thisx)
 
+#define STATE_STUNNED (1 << 0)
```

Comment:
For a similar reason, I suggest using a more specific prefix here because `STATE_` might conflict with the rest of the codebase in the future.